app/views/ukrentaroom/adjustments/RaRAdjustmentsIntroView.scala.html

```diff
@@ -0,0 +1,41 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.adjustments.routes
+
+@this(
+        layout: templates.Layout,
+        govukButton: GovukButton
+)
+
+@(taxYear: Int)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("raRAdjustmentsIntro.title"))) {
+
+    <h1 class="govuk-heading-l">@messages("raRAdjustmentsIntro.heading")</h1>
+
+<p class="govuk-body">@messages("raRAdjustmentsStart.para1")</p>
+
+<ul id="list" class="govuk-list govuk-list--bullet">
+ <li>@messages("raRAdjustmentsIntro.bullet.content.l1")</li>
+</ul>
+
+@govukButton(
+ButtonViewModel(messages("propertyIncomeStart.button"))
+.asLink(routes.PrivateUseAdjustmentController.onPageLoad(taxYear, NormalMode).url)
```

Comment:
correct link?

Reply:
Just a placeholder for now, will be corrected when I work on the CYA for this journey